en/sql/query/show.rst

```diff
@@ -1431,15 +1431,21 @@ Total_value                         INT             Total number of values store
 Avg_num_value_per_key               INT             Average number of values (OIDs) per key
 Num_leaf_page                       INT             Leaf page count
 Num_non_leaf_page                   INT             NonLeaf page count
+Num_ovf_page                        INT             OID overflow page count
```

Comment:
The description seems to be difficult for users to understand.
So, how about changing OID overflow to leaf's overflow?

Reply:
Received comments and edited.
Thank you.